schema.sql

```diff
@@ -24,4 +24,26 @@ CREATE TABLE species (
 
 ALTER TABLE animals
 ADD COLUMN species_id INTEGER,
-ADD COLUMN owner_id INTEGER;
\ No newline at end of file
+ADD COLUMN owner_id INTEGER;
+
+CREATE TABLE vets (
+    id INTEGER PRIMARY KEY AUTOINCREMENT,
+    name TEXT,
+    age INTEGER,
+    date_of_graduation DATE
+);
+
+CREATE TABLE specializations (
+  vet_id INTEGER REFERENCES vets(id),
+  species_id INTEGER REFERENCES species(id),
+  PRIMARY KEY (vet_id, species_id)
```

Comment:
[OPTIONAL]
- [ ] Since the columns of `specializations` table are meant to hold the references, there should be no need to give it a `primary key`. you consider deleting that.